packages/terra-clinical-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -10,6 +10,10 @@ import styles from './Cell.module.scss';
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  /**
+   * Accessible text provided for the Cell.
+   */
+  ariaLabel: PropTypes.string.isRequired,
```

Comment:
Is this always passed in?

Reply:
Not 100% sure, my assumption was that is was but maybe I should drop the `.isRequired` part.

Reply:
It shouldn't be required, only the RowSelectionCell provides it because it's the only Cell without custom content. Any other cell would provide aria attributes through its custom content.

Reply:
Updated in this commit: https://github.com/cerner/terra-clinical/pull/460/commits/10a6eafbc38572be48d4071d98ee59642aa7c840